DigitalLearningSolutions.Web.Tests/Controllers/Support/FaqControllerTests.cs

```diff
@@ -0,0 +1,193 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.Controllers.Support
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Models.Support;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.Controllers.Support;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.Support.Faqs;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.FeatureManagement;
+    using NUnit.Framework;
+
+    public class FaqControllerTests
+    {
+        // TODO HEEDLS-608 Review below tests to determine if any in here are excess to requirements,
+        // i.e. if the test is covered under HEEDLS-608 tests
+
+        private readonly IEnumerable<Faq> Faqs = new List<Faq>

```

Comment:
Minor naming warning I've only just spotted, should be "faqs".

Reply:
Done.